real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/GpsPanel.java

```diff
@@ -119,11 +169,8 @@ private void addCacheToTree() {
 			
 			FilenameFilter filter = (dir, name) -> {
                 // TODO Auto-generated method stub
-                if(name.contains("png")) {
-                    return true;
-                }
-                return false;
-            };
+				return name.contains("png");
```

Comment:
this method return type is void, so you really should not be returning a value.  also do something with the comment